DEVELOPMENT.md

```diff
@@ -30,6 +30,7 @@ To setup the environment manually you will need:
 
 - Just: [just.systems](https://just.systems/)
 - uv `>=0.4.27`: [docs.astral.sh](https://docs.astral.sh/uv/getting-started/installation/)
+    * If you have an older manually installed `uv` version, you can upgrade it with `uv self update`.
```

Comment:
I think if this is mentioned, we don't really need the script. Though might be nice to add the minimum version required in this line.